File: Baekjoon_Online_Judge/NO_02765/2765.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 20                   | 
 | 2765 자전거 속도           | 
 ---------------------------- 
*/
#define PI 3.1415927

int main(){
    int test_case_cnt = 1;

    while(true){
        double diameter, rotate_cnt, time, radius;

        cin >> diameter >> rotate_cnt >> time;

        if(rotate_cnt == 0) break;
        
        radius = diameter / 2 / (5280 * 12);
        
        double distance = 2 * PI * radius * rotate_cnt;
        double mph = distance / time * 3600;

        printf("Trip #%d: %.2lf %.2lf\n", test_case_cnt, distance, mph);

        test_case_cnt += 1;
    }
    return 0;
}

File: Baekjoon_Online_Judge/NO_02765/2765.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 20                   | #
# | 2765 자전거 속도           | #
# ---------------------------- #


if __name__ == "__main__":
    test_case_cnt = 1
    PI = 3.1415927

    while True:
        diameter, rotate_cnt, time = map(float, r_input().split())

        if rotate_cnt == 0:
            break

        radius = (diameter / 2) / (5280 * 12)

        distance = PI * radius * 2 * rotate_cnt
        
        print("Trip #" + str(test_case_cnt) + ': %.2f %.2f' % (distance, distance / time * 3600))
        
        test_case_cnt += 1
